### DISCUSSION TITLE
Need help with LineChart

### DISCUSSION THREAD

**Discussion by uzumakiabi** [vikramlearning/blazorbootstrap#516] [Q&A]
Does anyone now how can the Labels in LineChart be edited, there is option for LineChartDataset to be edited, but i cant find anything for LineCharDataLabel

**Comment by gvreddy04** [comment]
@uzumakiabi Pls check https://demos.blazorbootstrap.com/charts

**Comment by uzumakiabi** [comment]
I’ve already checked the demos, the documentation, and i still can’t find it. I can declare it the same way as LineChartData, but I can’t implement like chart data, I finished with work for the week. On Monday im gonna send you what I need, how I use chart data and what is problem.Thank you in advance, have a good day and the rest of the weekSent from my iPhoneOn 18 Jan 2024, at 15:28, Vikram Gaddam ***@***.***> wrote:﻿
@uzumakiabi Pls check https://demos.blazorbootstrap.com/charts

**Comment by uzumakiabi** [comment]
﻿I’ve already checked the demos, the documentation, and i still can’t find it. I can declare it the same way as LineChartData, but I can’t implement like chart data, I finished with work for the week. On Monday im gonna send you what I need, how I use chart data and what is problem.Thank you in advance, have a good day and the rest of the weekSent from my iPhoneOn 18 Jan 2024, at 15:28, Vikram Gaddam ***@***.***> wrote:﻿
@uzumakiabi Pls check https://demos.blazorbootstrap.com/charts

**Comment by uzumakiabi** [comment]
﻿﻿I’ve already checked the demos, the documentation, and i still can’t find it. I can declare it the same way as LineChartData, but I can’t implement like chart data, I finished with work for the week. On Monday im gonna send you what I need, how I use chart data and what is problem.Thank you in advance, have a good day and the rest of the weekSent from my iPhoneOn 18 Jan 2024, at 15:28, Vikram Gaddam ***@***.***> wrote:﻿
@uzumakiabi Pls check https://demos.blazorbootstrap.com/charts

**Comment by gvreddy04** [comment]
@uzumakiabi Pls attach a screenshot of the chart and highlight the label you want to change.

**Comment by uzumakiabi** [comment]
I need this labels to be straight vertical, and the font size to be smaller Sent from my iPhoneOn 18 Jan 2024, at 16:55, Vikram Gaddam ***@***.***> wrote:﻿
@uzumakiabi Pls attach a screenshot of the chart and highlight the label you want to change.
